tests/baselines/reference/expandoFunctionNestedAssigments.js

```diff
@@ -51,7 +51,8 @@ for(let f in (Foo.forIn = []) ){
     {
         Foo.fromForInBodyNested = 1;
     }
-}
+}
+

```

Comment:
Why so may white space differences?

Reply:
I've rolled back, and noticed some things not being caught correctly so I needed to ad some changes in between. 
Please first review this PR :  https://github.com/bloomberg/TypeScript/pull/140/files and after that this one.